src/components/Home/DescriptionBoxes.tsx

```diff
@@ -0,0 +1,43 @@
+// Edit span tags to white after bg change to black
+
+export default function DescriptionBoxes() {
+    return (
+      <div className = "flex flex-col h-[150vh] md:h-[100vh] max-w-[1100px] justify-between w-[80%] m-auto">
```

Comment:
It might be more optimal to use a CSS grid here, but flexbox still works, just might be a little awkward around breakpoints as you are using rem sizing for margins rather than fractions